playbooks/tasks/portal-health-check-disable.yml

```diff
@@ -1,22 +1,27 @@
 ---
-# Take portal out of load balancer by stopping healh check
-
-- name: Sleep before portal health check disabling
-  wait_for:
-    timeout: "{{ portal_health_check_disable_delay_secs }}"
-  delegate_to: localhost
+# Take portal out of load balancer by stopping health check
 
+# Check if the health check container is running.
 - name: Check health check container is running
   community.docker.docker_container_info:
     name: health-check
-  register: result
+  register: health_check_docker_container_result
 
-- name: Stop portal healh check to take portal out of load balancer
-  command: 'docker exec health-check cli/disable "{{ out_of_lb_message }}"'
-  become: True
-  become_user: "user"
-  when: result.exists and result.container is defined and result.container.State.Running
+# Marker the health check as disabled via the cli command
+- name: Stop portal health check to take portal out of load balancer
+  # NOTE: portal_action is defined by the playbook, i.e. for deploys,
+  # portal_action='portal-deploy'
+  command: 'docker exec health-check cli disable "{{ out_of_lb_message | default(portal_action) }}"'
```

Comment:
do we want to make sure that it worked by adding a check whether http://10.10.10.60:3100/health-check returns response json where "disabled" equals the reason we just set ?

this can be a follow up too, I'll approve this just in case anyway